### PR TITLE
Publish the cl_img_memory_management extension specification.

### PR DESCRIPTION
**Pull request by tomasz-platek**
Publish the cl_img_memory_management extension specification.

### CODE REVIEW

File: extensions/cl_img_memory_management.asciidoc [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+CY Cheng, Imagination Technologies. +
+Tomasz Platek, Imagination Technologies.
+
+== Notice
+
+Copyright (c) 2024 Imagination Technologies Ltd. All Rights Reserved.
+
+== Status
+
+Draft spec, NOT APPROVED!!
```
Comment (bashbaug) [inline extensions/cl_img_memory_management.asciidoc:30]: This should probably be "Final Draft" or "Complete" or "Shipping" or similar.

```suggestion
Complete
```

File: extensions/cl_img_memory_management.asciidoc [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+
+== Version
+
+Built On: {docdate} +
+Version: Major.Minor.Patch
```
Comment (bashbaug) [inline extensions/cl_img_memory_management.asciidoc:35]:
```suggestion
Version: 1.0.0
```

**Review by bashbaug — APPROVED** [review]

File: extensions/cl_img_memory_management.asciidoc (hunk 1 above)
Comment (tomasz-platek) [reply extensions/cl_img_memory_management.asciidoc]: I overlooked it, this should be "Final Draft", already fixed.

**Comment by bashbaug** [comment]
PR has been open for over a month with no additional comments.  Merging!

**Pull request merged**